### PR TITLE
Feature/347 1

### PR DESCRIPTION
**Pull request by shivenduverma-sap**
- Added Status of last e2e tests in Readme.md
- Scheduled e2e tests for an everyday run
- Added Passed, Failed, Skipped, Errors, total tests, etc. status to e2e tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -72,7 +77,8 @@ test: manifests generate fmt vet envtest ## Run tests.
 
 .PHONY: e2e
 e2e:
-	go test -v -count=1 -tags e2e ./e2e
+	GOBIN=$(LOCALBIN) go install gotest.tools/gotestsum@${TAG}
```
Comment (phoban01) [inline Makefile:80]: Could we move this to a dedicated makefile target? And then add that as a dependency of the `e2e`  target?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # ocm-controller
 
-[![REUSE status](https://api.reuse.software/badge/github.com/open-component-model/ocm-controller)](https://api.reuse.software/info/github.com/open-component-model/ocm-controller) [![e2e](https://github.com/open-component-model/ocm-controller/actions/workflows/e2e.yaml/badge.svg)](https://github.com/open-component-model/ocm-controller/actions/workflows/e2e.yaml)
+[![REUSE status](https://api.reuse.software/badge/github.com/open-component-model/ocm-controller)](https://api.reuse.software/info/github.com/open-component-model/ocm-controller) 
+
+[![e2e-tests workflow](https://github.com/open-component-model/ocm-controller/actions/workflows/e2e.yaml/badge.svg)](https://github.com/open-component-model/ocm-controller/actions/workflows/e2e.yaml)
```
Comment (phoban01) [inline README.md:5]: Can we keep the badges on a single line?

File: Makefile (hunk 1 above)
Comment (shivenduverma-sap) [reply Makefile]: Done

File: Makefile [hunk 3]
```diff
@@ -71,8 +77,8 @@ test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
 .PHONY: e2e
-e2e:
-	go test -v -count=1 -tags e2e ./e2e
+e2e: test-summary-tool ## Runs e2e tests
+	$(GOTESTSUM) --format testname -- -tags=e2e ./e2e
```
Comment (phoban01) [inline Makefile:81]:
```suggestion
	$(GOTESTSUM) --format testname -- -count=1 -tags=e2e ./e2e
```

We need to want to disabling caching to ensure the tests run each time

File: README.md (hunk 2 above)
Comment (shivenduverma-sap) [reply README.md]: Done

Comment (shivenduverma-sap) [reply README.md]: Done

**Review by phoban01 — COMMENTED** [review]
Looks good, just spotted that cache was-renabled. I think should leave it disabled. I've added a suggestion below.

File: Makefile (hunk 3 above)
Comment (shivenduverma-sap) [reply Makefile]: Done

**Review by phoban01 — APPROVED** [review]
Looks good, thanks @shivenduverma-sap. 

A small nit: please use more descriptive commit messages in the future and try not to reuse the same commit message for different commits; it makes it hard to anybody browsing the history to understand what happened 😄

**Comment by shivenduverma-sap** [comment]
Lets discuss the commit messages soon, not really happy with commits without ticket numbers and every small commit having a unique message, git repo should indicate the modified files due to a ticket when multiple people are actively modifying the files. Blame from file/PR should indicate why the change was made.

**Comment by phoban01** [comment]
Also please ensure all commits are signed. Ticket numbers can be added to the commit message body. I think we should try to standardize our commit message approach and use something like conventional commits.

**Pull request merged**